src/main/kotlin/io/openfuture/state/controller/ExceptionHandler.kt

```diff
@@ -21,7 +20,7 @@ class ExceptionHandler {
     @ResponseStatus(HttpStatus.BAD_REQUEST)
     @ExceptionHandler(MethodArgumentNotValidException::class)
     fun handleException(ex: MethodArgumentNotValidException): ErrorDto {
-        val fieldErrors = ex.bindingResult.fieldErrors.map { FieldErrorDto(it.field, it.defaultMessage) }
+        val fieldErrors = ex.bindingResult.fieldErrors.map { ErrorDto.FieldErrorDto(it.field, it.defaultMessage) }
```

Comment:
Oh, sorry mate. I didn't see that we are preparing `FieldErrorDto` out of the `ErrorDto`. Then it makes sense to have an independent class `FieldErrorDto`.
Sorry again.